### PR TITLE
Fix all flake8 violations for asv/benchmark.py

### PR DESCRIPTION
**Pull request by dorothykiz1**

### CODE REVIEW

File: asv/benchmark.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 import copy
 try:
     import cProfile as profile
-except:
+except Exception:
```
Comment (datapythonista) [inline asv/benchmark.py:44]: We shouldn't capture all exceptions here, just with `ImportError` is enough.

File: asv/benchmark.py [hunk 2]
```diff
@@ -574,7 +568,7 @@ def do_run(self):
 
     def do_profile(self, filename=None):
         def method_caller():
-            run(*params)
+            run(*params) # noqa see #1020
```
Comment (datapythonista) [inline asv/benchmark.py:571]: When using `noqa` we need to specify the exact error code we want to ignore. Also, if besides pointing to the issue you can write a very short sentence on what's the problem  and why this should be skipped, that would be useful

File: asv/benchmark.py (hunk 1 above)
Comment (dorothykiz1) [reply asv/benchmark.py:44]: let me add this too

**Comment by dorothykiz1** [comment]
@datapythonista this PR is ready for your review

**Review by datapythonista — COMMENTED** [review]
Lools good, thanks for the work on this @dorothykiz1. Are all the problems in this file fixed? Can we validate it in the CI, so we know the changes here are correct, and future changes to this file don't introduce more problems please?

**Comment by dorothykiz1** [comment]
@datapythonista thanks for your review let me  merge this with the other PR and add it the CI

**Pull request closed without merging**